### PR TITLE
Merge Stable into Master 12-16

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Merges stable into master

### CODE REVIEW

**Review by aamustapha — COMMENTED** [review]
I'm neutral, just curious

File: app/assets/v2/js/cart.js [hunk 1]
```diff
@@ -52,13 +52,14 @@ Vue.component('grants-cart', {
       comments: undefined,
       hideWalletAddress: true,
       AnonymizeGrantsContribution: false,
-      include_for_clr: true,
+      include_for_clr: false,
```
[the remark this answers is not in the document]
Comment (octavioamu) [reply app/assets/v2/js/cart.js:55]: Im not doing nothing here just merging stable into master, what ever changes here is what is already on stable

[the remark this answers is not in the document]
Comment (octavioamu) [reply app/assets/v2/js/cart.js:55]: https://github.com/gitcoinco/web/commits/stable

[the remark this answers is not in the document]
Comment (octavioamu) [reply app/assets/v2/js/cart.js:55]: here you can check that commit https://github.com/gitcoinco/web/commit/954c1748a95f8b97836015cc353d23cc9e66c7c8 
Was due some people complaining about gitcoin contributions being checked by default

**Pull request merged**